### PR TITLE
Old data adapter for Alpha Vantage

### PR DESCRIPTION
**Pull request by ricdezen**
Added a method to change the datetime of an alphavantage json file to match gmt time, and included dictionary with timezone for all known tickers (always "US/Eastern", wow).

Not much to say tbh, I tried this on a batch of old files and seemed to work alright. Give it a try.

### CODE REVIEW

File: av_downloader.py [hunk 1]
```diff
@@ -63,6 +65,7 @@ def download(self, symbol: str, interval='1min'):
                     ).strftime("%Y-%m-%d %H:%M:%S")
                 ] = v
             meta = dict()
+            meta['timezone'] = timezone
```
Comment (ricdezen) [inline av_downloader.py]: Wait no why is this here

File: av_downloader.py [hunk 2]
```diff
@@ -90,6 +93,15 @@ def __convert_to_gmt(*, date_time: datetime, zonename: str) -> datetime:
         return base.astimezone(GMT)
 
 
+    @staticmethod
```
Comment (ricdezen) [inline av_downloader.py]: And why in hell did this get screwed up

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OTRI-Unipd/history-downloader/pullRequest?prid=5421224&bid=17777701)

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OTRI-Unipd/history-downloader/pullRequest?prid=5421224&bid=17777701)

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OTRI-Unipd/history-downloader/pullRequest?prid=5421224&bid=17777701)

File: old_data_adapter.py [hunk 3]
```diff
@@ -20,29 +25,68 @@ def list_folders(folder : str):
...
+procedure = None
+while True:
+    provider_name = input("Who provided this data? y/a: ")
+    if(provider_name == "y"):
+        procedure = adapt_from_yf
+        break
+    if(provider_name == "a"):
+        procedure = adapt_from_av
+        break
+    print("You couldn't even choose between two letters, wow")
```
Comment (CremaLuca) [inline old_data_adapter.py:86]: Lovely

Comment (CremaLuca) [reply old_data_adapter.py:86]: Wait, was it me that added it or was it you?

Comment (CremaLuca) [reply old_data_adapter.py:86]: Oh, wait it was me. My past self made future myself blow air through my nose.
I'm impressed.

File: tz_dict.json [hunk 4]
```diff
@@ -0,0 +1,2536 @@
...
+    "RECN": "US/Eastern",
+    "RPXC": "US/Eastern",
+    "RTN": "US/Eastern",
+    "SGYP": "US/Eastern",
+    "SHLD": "US/Eastern",
+    "S": "US/Eastern",
+    "TAX": "US/Eastern",
+    "UTX": "US/Eastern",
+    "WAAS": "US/Eastern",
+    "WIN": "US/Eastern"
```
Comment (CremaLuca) [inline tz_dict.json:2535]: You were right, these are all actually US/Eastern

**Review by CremaLuca — APPROVED** [review]
Ok
